tests/yast2_gui/yast2_network_settings.pm

```diff
@@ -57,12 +58,14 @@ sub run {
     unless (is_network_manager_default) {
         send_key $cmd{overview_tab};
         assert_screen 'yast2-network-settings_overview';
+        wait_still_screen 2;
```

Comment:
I would like to use assert_screen_change { send_key xx } try make sure page open or send_key_until_needlematch.
My guess after assert_screen the screen should already still?

Reply:
I have different opinion on this, the 'add' button may not work at that point which cause send key but next screen not shown, so we need wait to make sure the 'add' button work then send key.

See the latest VR, I have run 10 times and no such issue found any more.

Reply:
assert_screen doesn't warranted that, the best way to fix that is with libyui-rest-api and we are not investing now on the scope of this ticket, so I guess this solution is fine.